plugins/module_utils/netbox_utils.py

```diff
@@ -501,7 +501,10 @@ def _validate_query_params(self, query_params):
         # Fetch the OpenAPI spec to perform validation against
         base_url = self.nb.base_url
         junk, endpoint_url = nb_endpoint.url.split(base_url)
-        response = open_url(base_url + "/docs/?format=openapi")
```

Comment:
Do you mind reverting this change? We don't use this method currently and `pynetbox` has a method we can use to do exactly what this function is doing.